polls/models.py

```diff
@@ -12,7 +12,8 @@ def __str__(self):
         return self.question_text
 
     def was_published_recently(self):
-        return self.pub_date >= timezone.now() - datetime.timedelta(days=1)
+        now = timezone.now()
+        return now - datetime.timedelta(days=1) <=self.pub_date <=now
```

Comment:
```suggestion
        return now - datetime.timedelta(days=1) <= self.pub_date <= now
```
Please use flake8 proceeding forward to check your code. Google on what flake8 is and how it should be used.